documentation/jetty-documentation/src/main/java/org/eclipse/jetty/docs/programming/server/http/HTTPServerDocs.java

```diff
@@ -779,12 +778,14 @@ public void multipleResourcesHandler() throws Exception
         ResourceHandler handler = new ResourceHandler();
 
         // For multiple directories, use ResourceCollection.
-        ResourceCollection directories = new ResourceCollection();
-        // TODO: how to add paths to ResourceCollection?
-//        directories.addPath("/path/to/static/resources/");
-//        directories.addPath("/another/path/to/static/resources/");
-
-        handler.setBaseResource(directories);
+        Resource.Mount mount = Resource.mountCollection(
```

Comment:
Since the `Resource`s could be created, there's no need for mounting.

Reply:
Greatly simplified